src/dashboard/messages.ts

```diff
@@ -0,0 +1,16 @@
+export default {
+    REGISTRATION_CANCELLED:                   'The setup process has been aborted.',
+    REGISTRATION_ENTER_EMAIL_INVITATION:      'Log into dashboard.testcafe.io to start the setup process.\nEnter your email to receive a login link.',
+    REGISTRATION_EMAIL_SENDING_NETWORK_ERROR: 'Error: cannot request the log-in email. Please check your internet connection.',
+    REGISTRATION_EMAIL_SENT:                  'Check your inbox for the TestCafe Dashboard log-in email. Follow the enclosed link to log into the application.\nCopy the project token and paste it here.',
```

Comment:
```suggestion
    REGISTRATION_EMAIL_SENT:                  'Check your inbox for the TestCafe Dashboard log-in email. Follow the link to open the Token Setup page.\nProject tokens are unique authorization keys that let you upload data to Dashboard. Copy the project token and paste it here.'',
```

also a break in logic here... why do we mention tokens out of the blue like that?